docs/csharp/language-reference/keywords/default-values-table.md

```diff
@@ -52,19 +34,23 @@ Beginning with C# 7.1, you can use the [`default` literal](../../programming-gui
 int a = default;
 ```
 
-You also can use the parameterless constructor or the implicit parameterless constructor to produce the default value of a value type, as the following example shows. For more information about constructors, see the [Constructors](../../programming-guide/classes-and-structs/constructors.md) article.
+For a value type, the implicit parameterless constructor also produces the default value of the type, as the following example shows:
```

Comment:
@BillWagner Do I understand that correctly: a value type cannot have a user-defined, that is, explicit, parameterless constructor, so any parameterless constructor is implicit?

Reply:
That's correct, @pkulikov.